### PR TITLE
Crud admin article

### PR DESCRIPTION
**Pull request by Monost0ne**
scss not good actually, just first iteration

### CODE REVIEW

File: assets/styles/admin_article.scss [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+    width: auto;
+}
+
+h1 {
+    margin-top: 30px;
+}
+
+form {
+    font-weight: bold;
+    font-family: 'Titillium Web', sans-serif;
```
Comment (LuckyShuii) [inline assets/styles/admin_article.scss:14]: isn't the font supposed to be globally applied throughout the project?

File: src/Controller/AdminArticleController.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    #[Route('/new', name: 'app_admin_article_new', methods: ['GET', 'POST'])]
+    public function new(Request $request, EntityManagerInterface $entityManager): Response
+    {
+        $article = new Article();
+        $form = $this->createForm(ArticleType::class, $article);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager->persist($article);
+            $entityManager->flush();
```
Comment (LuckyShuii) [inline src/Controller/AdminArticleController.php:34]: you forgot to set a creation date here

File: src/Form/ArticleType.php [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                ]
+            ])
+            ->add('homePreview', TextareaType::class, [
+                'label' => 'Description Homepage',
+                'required' => true,
+                'help' => 'Le texte de présentation présent sur la homepage.',
+                'constraints' => [
+                    new NotBlank([
+                        'message' => 'Veuillez renseigner la description pour la homepage',
+                    ]),
```
Comment (LuckyShuii) [inline src/Form/ArticleType.php:59]: anglicisme "homepage" => translate to french

File: src/Form/ArticleType.php [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                    new NotBlank([
+                        'message' => 'Veuillez renseigner le corps de votre article',
+                    ]),
+                ]
+            ])
+            ->add('status', ChoiceType::class, [
+                'label' => 'Statut',
+                'help' => 'Choix du statut de votre article.',
+                'required' => true,
+
```
Comment (LuckyShuii) [inline src/Form/ArticleType.php:94]: remove blank space here

File: templates/admin_article/_form.html.twig [hunk 5]
```diff
@@ -0,0 +1,93 @@
+{{ form_start(form) }}
+    <div>
+        <div>
+            {{ form_label(form.articleTitle) }}
+        </div>
+        {{ form_widget(form.articleTitle) }}
+        <div class="formHelper">
+            {{ form_help(form.articleTitle) }}
+        </div>
+    </div>
```
Comment (LuckyShuii) [inline templates/admin_article/_form.html.twig:10]: weird indentation

File: templates/admin_article/_form.html.twig [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+</div>
+<div>
+    <div>
+        {{ form_label(form.category) }}
+    </div>
+    {{ form_widget(form.category) }}
+    <div class="formHelper">
+        {{ form_help(form.category) }}
+    </div>
+</div><div>
```
Comment (LuckyShuii) [inline templates/admin_article/_form.html.twig:82]: <div> x2, put the second next line

File: templates/admin_article/_form.html.twig [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+        {{ form_label(form.imgSrc) }}
+    </div>
+    {{ form_widget(form.imgSrc) }}
+    <div class="formHelper">
+        {{ form_help(form.imgSrc) }}
+    </div>
+</div>
+
+    <button class="btn">{{ button_label|default('Save') }}</button>
+{{ form_end(form) }}
```
Comment (LuckyShuii) [inline templates/admin_article/_form.html.twig:93]: translate "save" to French  + are you really using the class "btn" that Symfony provided you?

File: templates/admin_article/new.html.twig [hunk 8]
```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}New Article{% endblock %}
+{% block stylesheets %}{{ encore_entry_link_tags('admin_article') }}{% endblock %}
```
Comment (LuckyShuii) [inline templates/admin_article/new.html.twig:4]: Translate title into French

**Review by LuckyShuii — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by LuckyShuii — APPROVED** [review]

**Pull request merged**